### PR TITLE
style:line width 100

### PR DESCRIPTION
**Pull request by patrinoua**
The type of this PR is: **style**
This PR resolves adding as a style line width of 100

Poll: https://artsy.slack.com/archives/C02BAQ5K7/p1642522846027100

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [x] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

### Changelog updates

ⓘ 'User-facing' changes will be published as release notes.

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Rename modal title - patrinoua)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e53c269f48df66c73a577700c3e1f06926887f88
</p>

File: src/lib/Components/Artist/ArtistInsights/ArtistInsightsAuctionResults.tsx [hunk 1]
```diff
@@ -108,19 +118,20 @@ const ArtistInsightsAuctionResults: React.FC<Props> = ({ artist, relay, scrollTo
...
-        These auction results bring together sale data from top auction houses around the world, including Christie’s,
-        Sotheby’s, Phillips and Bonhams. Results are updated daily.
+        These auction results bring together sale data from top auction houses around the world,
+        including Christie’s, Sotheby’s, Phillips and Bonhams. Results are updated daily.
       </Text>
       <Spacer mb={2} />
       <Text>
-        Please note that the sale price includes the hammer price and buyer’s premium, as well as any other additional
-        fees (e.g., Artist’s Resale Rights).
+        Please note that the sale price includes the hammer price and buyer’s premium, as well as
```
Comment (pvinis) [inline src/lib/Components/Artist/ArtistInsights/ArtistInsightsAuctionResults.tsx:126]: is this.... ok? 🤔

File: package.json [hunk 2]
```diff
@@ -315,7 +315,7 @@
     ]
   },
   "prettier": {
-    "printWidth": 120,
+    "printWidth": 100,
```
Comment (pvinis) [inline package.json:318]: 👋 120

File: src/lib/Components/Artist/ArtistInsights/ArtistInsightsAuctionResults.tsx (hunk 1 above)
Comment (pvinis) [reply src/lib/Components/Artist/ArtistInsights/ArtistInsightsAuctionResults.tsx:126]: confirmed its fine.

**Pull request merged**